src/it/scala/org/zalando/spearheads/innkeeper/routes/RoutesRepoHelper.scala

```diff
@@ -14,18 +15,29 @@ object RoutesRepoHelper extends DaoHelper {
     createdAt: LocalDateTime = LocalDateTime.now(),
     activateAt: LocalDateTime = LocalDateTime.now().minusHours(2)): RouteRow = {
 
+    val path = pathsRepo.insert(PathRow(
```

Comment:
you should extract the creating of the path to another private method, as it's not relevant for our test